hw/ip/otbn/dv/model/otbn_model_dpi.h

```diff
@@ -52,11 +59,9 @@ void otbn_model_destroy(OtbnModel *model);
 // the contents of DMEM from the ISS and inject them into the simulation
 // memory.
 //
-// If start is true, we start the model and then step once (as described
-// above).
+// If start is true, we start the model at start_addr and then step once (as
+// described above).
```

Comment:
Ahah! I think this might have been a merge conflict. `start_addr` is no more. Probably just drop this hunk?